solo/docker/watchdog.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2015 Spotify AB
+
+import socket
+import asyncore
+import time
+import sys
+import os
+
+PORT=33333
```

Comment:
Is it worth setting this as an env var, rather than having it static in the Python script?